### PR TITLE
Restore crystal hollows changes that were nuked by NBT Data to Profile Viewer Tabs and XP Bars PR

### PR DESCRIPTION
**Pull request by CraftyOldMiner**
#94 looks like it accidentally reverted #95. This un-reverts the revert.

### CODE REVIEW

**Review by romangraef — APPROVED** [review]
In the future, just rebase your branches instead of merging master into your branch, that should make stuff like this less likely to happen

**Review by IRONM00N — APPROVED** [review]

**Comment by CraftyOldMiner** [comment]
> In the future, just rebase your branches instead of merging master into your branch, that should make stuff like this less likely to happen

<TLDR is at the bottom>
While I prefer merging over rebasing in most cases, I like to learn if I'm wrong and investigate better ways of doing things that prevent future problems. Here's what I found when I did...

- The NotEnoughUpdates/NotEnoughUpdates repo is using squash merges. This means that my original PR (#95) became a single, new, commit once it was merged: e202e4ad. The NEU master branch at that commit looks good. This means that   merging during development on my branch wasn't a factor in the accidental deletion.

- The commits that deleted the code came from these rebase-related commits:  https://github.com/walkerselby/NotEnoughUpdates/commit/ea80e1be54881705a2163b9aa4ba2cf41541d05d and https://github.com/NotEnoughUpdates/NotEnoughUpdates/pull/94/commits/4cf437311ba63d7e53933716b84039e26bb0d1bf as part of #94. The changes in the branch for that PR didn't follow the one major rule of rebasing: [You must not rebase commits other people did.](https://lwn.net/Articles/328438/) Put another way on the same page: "Because after you've done a merge, you can no longer rebase you commits".

**TLDR**
The impact of rebasing vs. merging differs based on the code development phase:

- _Prior to PR (1 branch, 1 dev):_ It's personal preference. Rebasing, merging, and/or squashing commits doesn't make a difference in what others see as long as you clean up and make your changes a single commit before creating a PR.
- _Prior to PR (1 branch, 2+ devs):_ Don't rebase. The branch is no longer private code, rebasing messes up other devs working on the branch and makes merging code more costly & complex.
- _During PR:_ It's debatable. I personally prefer merging since the separate merge-commit makes it easy to investigate bad merges and it's much easier to do things that are horribly wrong with rebase. I also find it much easier to resolve all merge conflicts together instead of commit-by-commit, especially when I have a lot of local changes. FWIW, Atlassian says to ["avoid using git rebase after creating the pull request"](https://www.atlassian.com/git/tutorials/merging-vs-rebasing#the-golden-rule-of-rebasing).
- _Completed PRs:_ Doesn't matter for this repo. Since PRs are being squashed on merge each PR ends up as a single commit in the commit history.
